cmd/create/cluster/cmd_test.go

```diff
@@ -136,3 +138,39 @@ var _ = Describe("Validates OCP version", func() {
 		})
 	})
 })
+
+var _ = Describe("Validate cloud accounts", func() {
+
+	Context("build billing accounts", func() {
+		When("return the error or result", func() {
+			It("OK: Successfully gets contracts from cloudAccounts", func() {
+				mockCloudAccount := v1.NewCloudAccount().CloudAccountID("1234567").
+					Contracts(v1.NewContract().StartDate(time.Now()).EndDate(time.Now().Add(2)).
+						Dimensions(v1.NewContractDimension().Name("control_plane").Value("4")))
+				cloudAccount, err := mockCloudAccount.Build()
+				Expect(err).NotTo(HaveOccurred())
+				_, isContractEnabled := GetBillingAccountContracts([]*v1.CloudAccount{cloudAccount}, "1234567")
+				Expect(isContractEnabled).To(Equal(true))
+			})
+
+			It("OK: Successfully print contract details", func() {
+				mockContract, err := v1.NewContract().StartDate(time.Date(2023, 10, 12, 12, 12, 12, 12, time.UTC)).
+					EndDate(time.Date(2023, 10, 12, 12, 12, 12, 12, time.UTC)).
```

Comment:
Can we simplify the date creation from a string?

Reply:
Hmm I don't think it can be created from a string though. Startdate's value takes time.Time only.
<img width="686" alt="image" src="https://github.com/openshift/rosa/assets/26282541/b502e0e9-08ac-466e-ad07-3f5b595e0fb3">

Reply:
Take a look at time.Parse https://pkg.go.dev/time#Parse

Reply:
Used parse instead 👍